sql/src/main/java/org/elasticsearch/action/admin/indices/create/TransportBulkCreateIndicesAction.java

```diff
@@ -353,7 +357,8 @@ private void validateAndFilterExistingIndices(ClusterState currentState,
         }
     }
 
-    private Settings createIndexSettings(ClusterState currentState, List<IndexTemplateMetaData> templates) {
+    private Settings.Builder createIndexSettingsBuilder(ClusterState currentState,
```

Comment:
I wouldn't rename the method to reflect the return type, it's about what the method does (creating settings)